### DISCUSSION TITLE
[Development] Tool updates

### DISCUSSION THREAD

**Discussion by Dendraspis** [staxrip/staxrip#572] [General]
### Let's use [Tool Update Requests](https://github.com/staxrip/staxrip/wiki/Tool-Update-Requests) instead.

~~I thought we could start a new discussion for that purpose instead of flooding other issues.~~

**Comment by Dendraspis** [comment]
### after v2.1.8.4

- x264 by Patman
- x265 by Patman

@Patman86 made changes that work pretty well with the latest features and changes.

**Reply by stax76** [reply]
This is what I got so far:

- SMDegrain 3.1.2.111s
- mkvtoolnix 55 (mkvextract Unicode fix)
- MP4Box 1.1.0 DEV rev542 gcc10.2.0 Patman (AAC 7.1 fix)
- rav1e 0.5.0-alpha gcc10.2.0 Patman
- DGMPGDec 2.0.0.5 (large file handling fix)
- aomenc 2.0.2-1350-msvc1928 Patman
- x265 3.5_RC1+2+29-gcc10.2.0 Patman Mod
- x264 0.161.3048+25-gcc10.2.0 Patman Mod
- ffmpeg N-101392-gcc10.2.0 Patman

**Reply by Dendraspis** [reply]
Very nice. 
Was unsure after I saw that you created a separate commit for MKVToolNix. 😊

Seems like you prepare the next Stable.  🤓

**Reply by jessielw** [reply]
I'm ready to put the next stable to the test :)

**Comment by Dendraspis** [comment]
### after v2.1.8.5

- ~~License.txt is missing~~
- MP4Box ~~must be replaced with the last one used since v2.1.8.2 because of a bug (see #584) as long there is no fix for that~~

**Reply by stax76** [reply]
Most likely it has something to do with moving the source to the Source sub folder.

**Reply by Dendraspis** [reply]
Yeah, but it's missing in 7z file. So for the next version we have to pay attention and include it. 😊

**Reply by stax76** [reply]
It was missing in my bin folder for unknown reason. I've copied it now back from the root folder.

**Reply by Dendraspis** [reply]
I added a note related to MP4Box....

**Reply by stax76** [reply]
Where did you add the note?

We could use a Wiki page 'Tool Update Requests' to track which tools can be updated.

**Reply by Dendraspis** [reply]
Just at the top of the replies.  😁  Was just meant for the notifications.

Wiki might be better, because everybody can edit and discussions become very unhandy at a certain length.

**Reply by stax76** [reply]
I've created a page for it:

https://github.com/staxrip/staxrip/wiki#tools

And added a menu item with shortcut to the page in the apps dialog.

Whoever clicks links in the wiki should be careful and take a look in the revision history to ensure the links can be trusted.

**Reply by JJKylee** [reply]
I edited and put some requests on the [Tool Update Requests](https://github.com/staxrip/staxrip/wiki/Tool-Update-Requests) page.

**Reply by stax76** [reply]
Thank you, I also worked a bit on the Wiki today:

1. At the top of the home page there is a link for listing recent improvements:

    [Most recent Wiki improvements](https://github.com/staxrip/staxrip/search?o=desc&q=&s=updated&type=wikis)

2. New Contributions section for people interested to contribute.

    https://github.com/staxrip/staxrip/wiki#contributions
